pymeasure/adapters/usbtmc.py

```diff
@@ -0,0 +1,170 @@
+#
+# This file is part of the PyMeasure package.
+#
+# Copyright (c) 2013-2016 PyMeasure Developers
+#
+# Python USBTMC driver, lifted from python-usbtmc.
```

Comment:
What is the license for `python-usbtmc`? Can this code be included under the MIT License?